src/ui/public/vislib/__tests__/components/color.js

```diff
@@ -21,14 +24,19 @@ describe('Vislib Color Module Test Suite', function () {
     var color;
 
     beforeEach(ngMock.module('kibana'));
-    beforeEach(ngMock.inject(function (Private) {
+    beforeEach(ngMock.inject(function (Private, config) {
+      previousConfig = config.get('visualization:colorMapping');
+      config.set('visualization:colorMapping', {});
       seedColors = Private(require('ui/vislib/components/color/seed_colors'));
       getColors = Private(require('ui/vislib/components/color/color'));
-      MappedColors = Private(require('ui/vislib/components/color/mapped_colors'));
-      mappedColors = new MappedColors();
+      mappedColors = Private(require('ui/vislib/components/color/mapped_colors'));
       color = getColors(arr);
     }));
 
+    afterEach(ngMock.inject(function (config) {
```

Comment:
nitpick - kind of inconsistent here in that you use fat arrow syntax in other new code